scripts/install_packages.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 
 # Handle errors http://redsymbol.net/articles/unofficial-bash-strict-mode/
-# set -eo pipefail
-# IFS=$'\n\t'
+set -eo pipefail
+IFS=$'\n\t'
 
 echo "Start installing packages"
 cd /home/ubuntu/ourrevolution
 source /home/ubuntu/.virtualenvs/ourrevolution/bin/virtualenvwrapper.sh
-workon ourrevolution
+workon ourrevolution || true
```

Comment:
Why do we need `|| true` here? If environment isn't available, isn't something wrong?

Reply:
https://github.com/Our-Revolution/site/pull/275/files

Reply:
https://github.com/Our-Revolution/site/pull/275